guides/common/modules/snip_host-names-for-http-proxy.adoc

```diff
@@ -0,0 +1,14 @@
+[id="host-names-for-http-proxy_{context}"]
+
+[cols="2,1,1",options="header"]
+|====
+| Host name | Port | Protocol
+| subscription.rhsm.redhat.com | 443 | HTTPS
+| subscription.rhn.redhat.com | 443 | HTTPS
```

Comment:
I think noone uses the `rhn` name anymore (and it wasn't present in one of the tables anyway), so let's drop it.